src/main/java/codeu/model/store/basic/UserStore.java

```diff
@@ -125,6 +135,21 @@ public boolean isUserRegistered(String username) {
    */
   public void setUsers(List<User> users) {
     this.users = users;
+    for (User user : users) {
+        users.add(user);
```

Comment:
Make sure you're adding to `this.users`, because `users` is actually the parameter list.

Reply:
Thank you, fixed now!